### PR TITLE
chore(docs): fixing console errors and warnings

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Description of changes

Fixing all the warnings and errors that come up in our docs site:

* Add a mock Amplify config to appease Auth
* Fix our preloads of fonts
* Adds `data/image` to `img-src` because Geo code is being executed which tries to add a 1x1 webp image to test for webp support
* Fixes our expander use to quiet our orw

#### Description of how you validated changes

| Before | After |
| - | - |
| CleanShot 2023-02-22 at 14 43 34@2x | CleanShot 2023-02-22 at 14 42 59@2x |

Note: the 403 will go away in prod

Also the Authenticator will show a generic network error if you try to log in:

CleanShot 2023-02-22 at 14 51 14@2x

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by calebpollman — DISMISSED** [review]
LGTM

File: docs/src/pages/_document.page.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       ...ANALYTICS_CSP.all.frame,
       ...ANALYTICS_CSP.prod.frame,
     ].join(' ')} *.youtube.com;
-    img-src 'self' blob: ${[
+    img-src 'self' blob: data: ${[
```
Comment (reesscot) [inline docs/src/pages/_document.page.tsx:44]: Can we lock this down to that specific data image?

Comment (dbanksdesign) [reply docs/src/pages/_document.page.tsx:44]: Great question! Unfortunately, no. It is a schema and you can't put anything behind the `:`. In general it sounds like data schema is bad to allowlist, BUT it sounds like maybe it isn't so bad for `img-src` : https://security.stackexchange.com/questions/94993/is-including-the-data-scheme-in-your-content-security-policy-safe/95011#95011

We could root cause this and fix whatever Geo code has a side effect that is running, but that sounds like a lot of work.

**Review by ioanabrooks — DISMISSED** [review]
Thank you for doing this!

**Review by reesscot — APPROVED** [review]
LGTM

**Review by calebpollman — APPROVED** [review]

**Pull request merged**